resources/slack-openapi-patched.json

```diff
@@ -1319,6 +1319,9 @@
                     "items": {
                         "additionalProperties": true,
                         "properties": {
+                            "id": {
+                                "type": "integer"
+                            },
```

Comment:
This should be done via a patch file: https://github.com/jolicode/slack-php-api/blob/master/doc/updating-sdk.md